docs/user-guide/src/bmo/ironic_installation.md

```diff
@@ -0,0 +1,136 @@
+# Install Ironic
+
+Baremetal Operator must be able to interact with Ironic. First,
+we will see what are the Ironic optional containers we need to
+run. After that we will go through Ironic installation in-cluster and
+out-of-cluster. In any case, there are a couple of containers that need
+to run in order to provision baremetal nodes.
+
+- ironic
+- ironic-inspector
+- ironic-endpoint-keepalived
+- ironic-log-watch
+- ipa-downloader
+- dnsmasq
+- httpd
+
+To know more about each container's functionality check the documentation
+[here](https://github.com/metal3-io/ironic-image#description).
+
+## Prerequisites
+
+Container runtime (e.g., docker, podman). Here we use docker.
+
+## Environmental variables
+
+<!-- include these inline so we can style an match variables -->
+{{#include ironic_variables.md}}
+
+## Optional containers
+
+In some circumstances, it might be necessary to install some additional containers on
+besides the mandatory Ironic containers. And this might depend on whether your target nodes are
```

Comment:
Can we make it clear and provide a list of mandatory and list of optional containers at the beginning since we mentioned it there as well, to avoid confusion about which container belongs to which list?

Reply:
I removed this section, we can add a section later on for development environment